src/duckling/canvas/_toolbars/canvas-scale.component.scss

```diff
@@ -1,5 +1,9 @@
+:host {
+    display: flex;
+    align-items: center;
+}
+
 .scale-label {
     width: 36px;
```

Comment:
This isn't wide enough for 3-digit percentages, so the hover effect of the plus button underlaps the % symbol. 
If you increase to 48px it works